gcassert.go

```diff
@@ -158,11 +160,22 @@ func GCAssert(w io.Writer, paths ...string) error {
 	// Next: invoke Go compiler with -m flags to get the compiler to print
 	// its optimization decisions.
 
-	args := []string{"build", "-gcflags=-m=2 -d=ssa/check_bce/debug=1"}
-	for i := range paths {
-		args = append(args, "./"+paths[i])
+	var args []string
+	var cmd *exec.Cmd
+
+	if useBazel {
+		args = []string{"build"}
+		for i := range paths {
+			args = append(args, strings.TrimPrefix(paths[i], "./"))
```

Comment:
Why do we trim `./` prefix here in bazel and no longer add this prefix in non-bazel case?

Reply:
Bazel expects package paths without `./` so that’s why we trim it. For the non-bazel case, we never added it here, it was expected that the input path is prefixed with `./` because `parseDirectives` expects it in that way.

Reply:
We had
```
args = append(args, "./"+paths[i])
```
which does seem like we added this prefix for every path.

Reply:
I see, thanks for the explanation.

> maybe we should assert that all paths are prefixed with `./`

This idea sounds good to me - do you mind adding this assertion into this PR please?

Reply:
Sure, done.